css-forms-1/Overview.bs

```diff
@@ -385,11 +394,35 @@ Markup Shorthands: css yes, markdown yes
       With ''appearance: textfield'', the user agent must not generate this part.
   </dl>
 
-  ''::field-text'' and ''::clear-icon'' must be siblings.
+  ''::placeholder'', ''::field-text'' and ''::clear-icon'' must be siblings.
 
   ISSUE: Collect parts used by autofill.
 
-  ISSUE: Define something for the password visibility toggle for user agents implementing it?
+  ISSUE(11845): Define something for the password visibility toggle for user agents implementing it?
+
+## Styling Parts for textareas: the ''::placeholder'' and ''::field-text'' pseudo-elements ## {#textarea-pseudos}
+
+  <dl export>
+    <dt><span>''::placeholder''</span>
+    <dd>
+      The ''::placeholder'' pseudo-element represents
+      the portion of the <{textarea}> that contains the placeholder text.
+
+    <dt><span>''::field-text''</span>
+    <dd>
+      The ''::field-text'' pseudo-element represents
+      the portion of the <{textarea}> that contains the editable text.
+  </dl>
+
+  Their structure is defined as follows:
+
+    ```
+        <textarea>
+        ├─ ::placeholder
+        └─ ::field-text
+    ```
```

Comment:
Not convinced about this yet, can you reference https://github.com/w3c/csswg-drafts/issues/11844 instead?